modules/images-configuration-registry-mirror.adoc

```diff
@@ -84,10 +84,15 @@ spec:
   - mirrors:
     - mirror.example.com/redhat
     source: registry.redhat.io/openshift4 <3>
+  - mirrors:
+    - mirror.example.com <4>
+    source: registry.redhat.io <5>
 ----
 <1> Indicates the name of the image registry and repository.
 <2> Indicates the registry and repository containing the content that is mirrored.
 <3> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the `ImageContentSourcePolicy` resource is applied to all repositories from the registry.
+<4> Indicates the name of the mirror.
+<5> Indicates the name of the registry.
```

Comment:
Could we add those content like in https://bugzilla.redhat.com/show_bug.cgi?id=2056385#c2 ?
```
source: registry.example.com/foo/bar/myapp
mirrors: mirror.example.net/app
-> result: mirror.example.net/app@sha256:...

source: registry.example.com/foo/bar
mirrors: mirror.example.net
-> result: mirror.example.net/myapp@sha256:...

source: registry.example.com
mirrors: mirror.example.net/registry-example-com
-> result: mirror.example.net/registry-example-com/foo/bar/myapp@sha256:...

source: registry.example.com/foo
mirrors: mirror.example.net/mirrored-foo
-> result: mirror.example.net/mirrored-foo/bar/myapp@sha256:...
```

Reply:
Hi @xenolinux ,Please update per your understand, we can update registryrepomirror.yaml, such as
```
spec:
  repositoryDigestMirrors:
  - mirrors:
    - example.io/example/ubi-minimal <1>
    - example.com/example/ubi-minimal <2>
    source: registry.access.redhat.com/ubi8/ubi-minimal <3>
  - mirrors:
    - mirror.example.com/redhat
    source: registry.redhat.io/openshift4 <4>
  - mirrors:
    - mirror.example.com
    source: registry.redhat.io <5>
  - mirrors:
    - mirror.example.net/image
    source: registry.example.com/example/myimage <6>
  - mirrors:
    - mirror.example.net
    source: registry.example.com/example <7>
  - mirrors:
    - mirror.example.net/registry-example-com
    source: registry.example.com <8>
```
<2>  Identify multiple mirrored repositories for each target repository, to make sure that if one mirror is down, another can be used
<5> You can only configure the name of registry, then ImageContentSourcePolicy resource is applied to all repositories from source registry to mirror registry, the namespace should be the same.
<6> mirror.example.net/image@sha256:... will be pulled
<7>myimage under example namespace of source registry will be pulled from mirror.example.net/myimage@sha256:...
<8>ImageContentSourcePolicy resource is applied to all repositories of source registry to mirror.example.net/registry-example-com, registry.example.com/example/myimage will be pulled from mirror.example.net/registry-example-com/example/myimage@sha256:...